### ISSUE TITLE
Quantitative evaluation for texture

### ISSUE THREAD

**Issue by RockyXu66** [Skype-line/X-Avatar#20]
I notice the `color_loss` during training. However, I don't see any quantitative evaluation for textures in your paper, or the code. I also can't find any texture evaluation for other methods, such as [SCALE](https://qianlim.github.io/SCALE.html) and [SCANimate](https://scanimate.is.tue.mpg.de/). Do you have any idea why this is? I believe that the quality of texture colors could influence the final result, especially regarding facial details like teeth and cheeks.

**Comment by MoyGcc** [comment]
Yes, since prior works don't really evaluate the texture, we just follow their protocol and focus on the geometry side only. I think the reason might be that for all these scan-based methods, having a texture field on top of the human geometry isn't the main contribution of the methods and it might look a bit trivial. Evaluating the texture doesn't really help to highlight the focus of the paper itself. 

But I totally agree that the most fine-grained human surface details like the mouth and teeth are highly related to the texture and it's worth exploring leveraging the texture information to refine the geometry in turn.

**Comment by RockyXu66** [comment]
> Yes, since prior works don't really evaluate the texture, we just follow their protocol and focus on the geometry side only. I think the reason might be that for all these scan-based methods, having a texture field on top of the human geometry isn't the main contribution of the methods and it might look a bit trivial. Evaluating the texture doesn't really help to highlight the focus of the paper itself.
> 
> But I totally agree that the most fine-grained human surface details like the mouth and teeth are highly related to the texture and it's worth exploring leveraging the texture information to refine the geometry in turn.

Thanks for your reply :)

I am confused about "leveraging the texture information to refine the geometry in turn". Does this mean that we might improve the accuracy of the geometry by using 2D rendered images, such as [PointAvatar](https://zhengyuf.github.io/PointAvatar/)? However, I am considering that scan-based dataset contain 3D information, which is generally more efficient for learning compared to 2D images. In that case, what are the advantages of using 2D information for refinement purposes?

**Comment by MoyGcc** [comment]
Yes, 3D information in scans is more direct for learning 3D models compared to 2D images. However, in most use cases, we don't have 3D scans of the target subjects, and in general 3D/4D assets are still very limited. On the other hand, the 3D assets are indeed not that perfect because they are also obtained from multi-view 2D images via some traditional vision and graphics algorithms which can introduce errors unless we input manual efforts.

In contrast, 2D images themselves barely contain errors or inaccurate information. Learning from these resources would ultimately reach higher quality and is practically more useful.

**Comment by RockyXu66** [comment]
I agree with you. So, there is no point in introducing 2D images projected from 3D scans for this representation (SNARF), right? What you are talking about is only using 2D images as training dataset to improve the fidelity of the constructed 3D avatar?

**Comment by MoyGcc** [comment]
I didn't really try using 2D information as an objective to supervise the SNARF learning, but I think doing so is indeed mutually beneficial, i.e., 2D can compensate information that 3D scans don't have or lost, and 3D scans, as a warm up for avatar learning, can be very reliable compared to learning humans from 2D images only. The easiest way I can come up with is to simply leverage UNISURF (occ-based volume rendering) on top of Fast-SNARF and that could quickly do a proof of concept.

**Comment by RockyXu66** [comment]
Thanks. I will have a try.
